src/components/filterPatterns/dialog/index.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { FC } from 'react';
+import {
+  Dialog as ChakraDialog,
+  Portal,
+  useDisclosure,
+  useSlotRecipe,
+} from '@chakra-ui/react';
+
+import { TranslationProvider } from 'src/components/translationProvider';
+
+import { FilterHeading } from '../Heading';
+import FilterActionButton from '../ActionButton';
+
+import { DialogFilterProps } from './props';
+import { OpenFilterButton } from './OpenFilterButton';
+
+export const DialogFilter: FC<DialogFilterProps> = ({
+  actionButton,
+  displayValue,
+  initialDialogState = 'closed',
+  Icon,
+  isApplied,
+  label,
+  language,
+  numberOfAppliedFilters,
+  onDialogClose,
+  onDialogOpen,
+  onResetFilter,
+  showCallToActionButton = true,
+  header,
+  isDisabled = false,
+  children,
+  paddingX,
+  backgroundColor,
+}) => {
+  const { onOpen, onClose, open } = useDisclosure({
+    defaultOpen: initialDialogState === 'open',
+    onOpen: onDialogOpen,
+    onClose: onDialogClose,
+  });
+
+  const recipe = useSlotRecipe({ key: 'dialogFilter' });
+  const [recipeProps] = recipe.splitVariantProps({});
+  const styles = recipe(recipeProps);
```

Comment:
I am wondering, if we do not have defined variant/size props in slotRecipes, do we still want to use `splitVariantProps` function or we can extract styles using `recipe()` function only. WDYT?

```suggestion
  const styles = recipe()
```

Reply:
Yes! Thank you!